universal-login-react/src/ui/TopUp/ChooseTopUpMethod.tsx

```diff
@@ -36,7 +38,7 @@ export const ChooseTopUpMethod = ({contractAddress, onPayClick, topUpClassName,
                 onClick={() => setTopUpMethod('crypto')}
                 checked={topUpMethod === 'crypto'}
                 name="top-up-method"
-                className={`top-up-method ${topUpMethod === 'crypto' ? 'active' : '' }`}
+                className={`top-up-method ${topUpMethod === 'crypto' ? 'active' : ''}`}
```

Comment:
extract `activeClassFor(topUpMethod === 'crypto', 'active')`
@alexcherman 
In one of following PRs

Reply:
ok